src/chrome/content/rules/Cp.pt.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Cp.pt">
+  <target host="cp.pt" />
```

Comment:
This is not a secure redirect this way.
You need to add the following line before the generic redirect:
```<rule from="^http://cp\.pt/" to="https://www.cp.pt/" />```